nucypher/blockchain/eth/sol/source/contracts/Adjudicator.sol

```diff
@@ -138,14 +126,14 @@ contract Adjudicator is Upgradeable {
         // A task specification is: capsule + ursula pubkey + alice address + blockhash
         bytes32 stampXCoord;
         assembly {
```

Comment:
could you explain what the next few lines do and why it can't be achieved in vanilla solidity?

Reply:
Something from old days, I hope @cygnusv can help

Reply:
I think this was just trying to get a slice of the array. We started this code when Solidity was v0.4 or v0.5, but it seems they introduced slices for calldata arrays in version 0.6. Maybe worth considering improving this part! What do you think @vzotova @theref ?

Reply:
My opinion is that Adjudicator is not worth time rn